pkg/reconciler/knativeserving/ingress/ingress_test.go

```diff
@@ -21,52 +21,51 @@ import (
 	"os"
 	"testing"
 
+	mf "github.com/manifestival/manifestival"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes/scheme"
-
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-
-	mf "github.com/manifestival/manifestival"
+	"k8s.io/client-go/kubernetes/scheme"
 	servingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
 	"knative.dev/operator/pkg/reconciler/common"
 	util "knative.dev/operator/pkg/reconciler/common/testing"
 )
 
-const numberIngressResource = 27
-
 func TestGetIngress(t *testing.T) {
 	os.Setenv(common.KoEnvKey, "testdata/kodata")
 	defer os.Unsetenv(common.KoEnvKey)
 
 	tests := []struct {
-		name                 string
-		targetVersion        string
-		expected             bool
-		expectedResourcesNum int
+		name                string
+		version             string
+		expectedNoErr       bool
```

Comment:
I personally like `expectError` instead of `expectedNoErr`. Because it is simple and we can omit `false` as it is default.

Reply:
I will change the `expectError` into a real error, instead of the bool.
If it is nil, check the equality. if it is an error, check the string of the error.